src/main.rs

```diff
@@ -191,10 +191,32 @@ impl State {
         let crt = tbs.sign(&pki)?;
         Ok(Self { key, crt, san })
     }
+
+    #[cfg(target_arch = "wasm32")]
```

Comment:
This is only used for testing and does not belong in production code

Reply:
Where should it go then? Maybe a decorator to only compile it for testing? Also, this addresses today's lack of file system support in Enarx.

Reply:
Resolved, no test data is ever inadvertently added to production code.